shared/profile/post-proof-container.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, {Component} from 'react'
+import PostProof from './post-proof'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import {cancelAddProof, checkProof, outputInstructionsActionLink} from '../actions/profile'
+
+class PostProofContainer extends Component<void, any, void> {
+  static parseRoute (currentPath, uri) {
```

Comment:
We could have a function that does this pattern of making a parseRoute for a given component, then we can just connect the dumb component and don't need this useless wrapper.

(Just a thought, no work necessary for this comment)